flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -237,7 +240,7 @@ private void testKeyedValueStateUpgrade(
     //  Tests for keyed ListState
     // -------------------------------------------------------------------------------
 
-    @Test
+    @TestTemplate
     public void testKeyedListStateMigration() throws Exception {
```

Comment:
```suggestion
    void testKeyedListStateMigration() throws Exception {
```
Could be package private